robottelo/ui/discoveryrules.py

```diff
@@ -65,8 +52,11 @@ def search(self, name):
         """Searches existing discovery rule from UI. It is necessary to use
         custom search as we don't have both search bar and search button there.
         """
+        # Remove entire method once BZ 1233135 is fixed
```

Comment:
This means that the search method should be removed when that BZ is fixed?

I recommend adding some type of validation based on that BZ to make sure we don't forget to update this. That said bz_bug_is_open can be used here.